alacritty/src/migrate.rs

```diff
@@ -60,6 +62,7 @@ fn migrate_config(
     options: &MigrateOptions,
     path: &Path,
     recursion_limit: usize,
+    warn_pruned: bool,
```

Comment:
Rather than adding another option here, we should probably just use the `options.dry_run` option to ensure consistency with other code. That way we also make sure to **always** warn on the dry run, which ensures that even on failure everything is logged.